schemas/parent.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: parent
+  title: Parent
+  group: 2
+  short: These fields contain information about a parent.
+  description: >
+    These fields contain information about a parent.
+
+    These fields provide more context about the parent process
+    and thread that are related to the data in the document.
+  type: group
+  fields:
+    - name: process
```

Comment:
Is this a breaking change? did `process.parent` -> `parent.process`?

Reply:
Yes

Reply:
This doesn't seem like a necessary breaking change. Outside of potentially simplifying some of this code base, this breaks a ton of detections for no added value, so I think it'd make more sense for use to figure out a more flexible way to keep the `process.parent` field set.

Reply:
@marshallmain can you explain your thought process for making a breaking change and why you think it's necessary?